pkg/ccl/backupccl/backup_test.go

```diff
@@ -1498,6 +1501,9 @@ func TestBackupRestoreResume(t *testing.T) {
 
 	params := base.TestClusterArgs{ServerArgs: base.TestServerArgs{
 		Knobs: base.TestingKnobs{JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals()},
+
+		// Cannot run this test within tenant because of the TestingGetPublicTableDescriptor func
```

Comment:
Is it because `TestingGetPublicTableDescriptor` doesn't take a codec? If so can we refactor it to. Otherwise lets add some more colour to this comment about what needs to change to enable this test.